src/core/Table/columns/actionColumn.tsx

```diff
@@ -0,0 +1,93 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { HeaderProps } from 'react-table';
+import { Checkbox } from '../../Checkbox';
+import SvgColumnManager from '@itwin/itwinui-icons-react/cjs/icons/ColumnManager';
+import { DropdownMenu } from '../../DropdownMenu';
+import { IconButton } from '../../Buttons/IconButton';
+import { MenuItem } from '../../Menu';
+import { tableResizeStartAction } from '../Table';
+import { EXPANDER_CELL_ID, SELECTION_CELL_ID } from '../hooks';
+
+const ACTION_CELL_ID = 'iui-table-action';
+
+export const ActionColumn = <T extends Record<string, unknown>>({
+  columnManager = false,
```

Comment:
Even though you set `columnManager` to false, if its not passed, but whole object is required to pass, so if i want to use `ActionColumn` without it, I need to write like that:
![image](https://user-images.githubusercontent.com/81580355/162165279-148897e7-bc3e-4777-8215-6be4b9cd592d.png)

So can make object optonal and.. if you had test with such a case, you would have caught it :D so add it

Reply:
Object is now optional. Test is written to for empty action column